### ISSUE TITLE
Fails to calculate Cairo 0 class hash when `program.attributes` is missing `accessible_scopes`

### ISSUE THREAD

**Issue by omerfirmak** [xJonathanLEI/starknet-rs#533]
Example contract triggering the error;

https://external.integration.starknet.io/feeder_gateway/get_class_by_hash?classHash=0xe91c3a17cd057ced2c2b371b914589e697b9558492e59e15a13cb4cebac63e

**Comment by xJonathanLEI** [comment]
Interesting. Didn't even know this was possible.

**Comment by xJonathanLEI** [comment]
Just tested with this command:

```console
starkli class-by-hash 0xe91c3a17cd057ced2c2b371b914589e697b9558492e59e15a13cb4cebac63e --parse
```

Indeed fails with: ``missing field `accessible_scopes` at line 1 column 92``.

**Comment by xJonathanLEI** [comment]
Just checked. Fortunately, `accessible_scopes` is not hashed when it's empty in the first place. So the fix here is simple, we just have to make deserialization interpret a missing field as `vec![]`.

**Comment by xJonathanLEI** [comment]
The fix has been back-ported to `starknet-core` `v0.8.x` as `v0.8.1`. IIRC Juno specifies its dependency version as `0.8.0`, which automatically resolves to `v0.8.1` as long as dependencies are not locked, which should be the case for Juno. That is to say, newer builds of Juno should automatically incorporate this fix.
